### ISSUE TITLE
Unhandled Runtime Error when loading Store or Marketplace pages

### ISSUE THREAD

**Issue by cinjoff** [onflow/kitty-items#211]
**Describe the bug**
Arriving on `Store` / `Marketplace` or after minting an NFT I get `Unhandled Runtime Error`

**To Reproduce**
- Start testnet based on README.MD instructions
- Initialized the service account manually using CLI as described here https://github.com/onflow/kitty-items/issues/164#issuecomment-1010432246 
- Visited `http://localhost:3001`

**Expected behavior**
See an empty store / no error

**Screenshots/Logs/Output**
```
Unhandled Runtime Error

Error: 3 errors occurred:
	* rpc error: code = Internal desc = failed to execute the script on the execution node execution-5f6c73a22445d7d958c6a37c1f3be99c72cacd39894a3e46d6647a9adb007b4d@execution-001.devnet32.nodes.onflow.org:3569=100: rpc error: code = Internal desc = failed to execute script: failed to execute script at block (4774e3eab8fb2b123d8b96a0d182711dc252653c1c708ee253906476a66ff6ae): [Error Code: 1101] cadence runtime error Execution failed:
error: cannot find declaration `MetadataViews` in `b30baffdcfa4ce48.MetadataViews`
 --> 8672ce4d3a0f54a667715b2ecbac30e7105ac66b80ea5b83516b25988234e7fe:2:7
  |
2 | import MetadataViews from 0xb30baffdcfa4ce48
  |        ^^^^^^^^^^^^^ available exported declarations are:


error: cannot infer type parameter: `T`
  --> 8672ce4d3a0f54a667715b2ecbac30e7105ac66b80ea5b83516b25988234e7fe:43:37
   |
43 |       if let view = item.resolveView(Type<MetadataViews.Display>()) {
   |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

	* rpc error: code = Internal desc = failed to execute the script on the execution node execution-a1f059a3e00347e585658e4c400be1580ba2e2f6d1f39387ad62e33f3f4d246d@execution-002.devnet32.nodes.onflow.org:3569=100: rpc error: code = Internal desc = failed to execute script: failed to execute script at block (4774e3eab8fb2b123d8b96a0d182711dc252653c1c708ee253906476a66ff6ae): [Error Code: 1101] cadence runtime error Execution failed:
error: cannot find declaration `MetadataViews` in `b30baffdcfa4ce48.MetadataViews`
 --> 8672ce4d3a0f54a667715b2ecbac30e7105ac66b80ea5b83516b25988234e7fe:2:7
  |
2 | import MetadataViews from 0xb30baffdcfa4ce48
  |        ^^^^^^^^^^^^^ available exported declarations are:


error: cannot infer type parameter: `T`
  --> 8672ce4d3a0f54a667715b2ecbac30e7105ac66b80ea5b83516b25988234e7fe:43:37
   |
43 |       if let view = item.resolveView(Type<MetadataViews.Display>()) {
   |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

	* rpc error: code = Internal desc = failed to execute the script on the execution node execution-48601e8a1568f18ca4a574a656710342857f939f598679280a6941cc7b800b6f@execution-003.devnet32.nodes.onflow.org:3569=100: rpc error: code = Internal desc = failed to execute script: failed to execute script at block (4774e3eab8fb2b123d8b96a0d182711dc252653c1c708ee253906476a66ff6ae): [Error Code: 1101] cadence runtime error Execution failed:
error: cannot find declaration `MetadataViews` in `b30baffdcfa4ce48.MetadataViews`
 --> 8672ce4d3a0f54a667715b2ecbac30e7105ac66b80ea5b83516b25988234e7fe:2:7
  |
2 | import MetadataViews from 0xb30baffdcfa4ce48
  |        ^^^^^^^^^^^^^ available exported declarations are:


error: cannot infer type parameter: `T`
  --> 8672ce4d3a0f54a667715b2ecbac30e7105ac66b80ea5b83516b25988234e7fe:43:37
   |
43 |       if let view = item.resolveView(Type<MetadataViews.Display>()) {
   |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Environment (Optional - Version, OS, Browser, etc.)**
macOS, Docker ran with `npm run start:testnet`, Firefox latest version

**Additional context (Optional - Anything else about the bug)**

**Comment by cinjoff** [comment]
**Update:**

After carefully reading through all the .cdc, I came to the conclusion that the MetadataView contract is being looked for under my account, and since the `flow.testnet.json` doesn't specify deployment of this contract, there wasn't one. 
The rest of the code suggests that this contract is already deployed on testnet, as it is referenced in the `flow.json` file, but when the application is ran, it seems to not look for it there, but rather under the account I setup as "admin".

So I did 2 changes.
1. Added `NFTStorefront` and `MetadataViews` to the list of `deployments.testnet` in my `flow.testnet.json` file.
2. Updated the NFTStoreFront reference for `testnet` which was specified in `flow.json` to point to my own account

With this, everything is running fine on testnet now. 

I am not resolving this issue yet, as I am not confident that this is the right way of me fixing stuff, so would like to get a confirmation first on this.
